test/controllers/SchemesOverviewControllerSpec.scala

```diff
@@ -37,16 +38,33 @@ import views.html.schemesOverview
 import scala.concurrent.Future
 
 class SchemesOverviewControllerSpec extends ControllerSpecBase with MockitoSugar with BeforeAndAfterEach {
- import SchemesOverviewControllerSpec._
+
+  import SchemesOverviewControllerSpec._
 
   val fakeCacheConnector: UserAnswersCacheConnector = mock[MicroserviceCacheConnector]
   val fakePsaMinimalConnector: MinimalPsaConnector = mock[MinimalPsaConnector]
   val appConfig: FrontendAppConfig = app.injector.instanceOf[FrontendAppConfig]
+
+  private def featureSwitchManagementService(isVariationsEnabled:Boolean):FeatureSwitchManagementService =
```

Comment:
cant we use the reusable featureSwitchManagementService test object

Reply:
Done